packages/botonic-cli/src/commands/deploy.ts

```diff
@@ -34,26 +34,40 @@ Uploading...
     }),
     command: flags.string({
       char: 'c',
-      description: 'Command to execute from the package "scripts" object'
+      description: 'Command to execute from the package "scripts" object',
+    }),
+    email: flags.string({
+      char: 'e',
+      description: 'Email from Hubtype Organization',
+    }),
+    password: flags.string({
+      char: 'p',
+      description: 'Password from Hubtype Organization',
+    }),
+    botName: flags.string({
+      char: 'b',
+      description: 'Name of the bot from Hubtype where you want to deploy',
     }),
-    botName: flags.string()
   }
 
   static args = [{ name: 'bot_name' }]
 
   private botonicApiService: BotonicAPIService = new BotonicAPIService()
+  private botName: string | undefined = undefined
 
   async run() {
     const { args, flags } = this.parse(Run)
     track('Deployed Botonic CLI')
 
     force = flags.force ? flags.force : false
     npmCommand = flags.command
-    let botName = flags.botName ? flags.botName : false
-
-    if (!this.botonicApiService.oauth) await this.signupFlow()
-    else if (botName) {
-      this.deployBotFromFlag(botName)
+    this.botName = flags.botName ? flags.botName : undefined
+    let email = flags.email ? flags.email : undefined
+    let password = flags.password ? flags.password : undefined
```

Comment:
same here as Dani suggest above. I'm wondering if we could update @botonic/cli to latest typescript versions to use the newest features/syntax as for example https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

Reply:
@vanbasten17 good point, actually cli already uses 3.7!
actually I'm not use why @guinii converts the empty values to undefined

Reply:
Both of you are right! I change the declaration! Thanks guys